app/controllers/partnering_organizations_controller.rb

```diff
@@ -52,6 +51,11 @@ def create
   # PATCH/PUT /partnering_organizations/1.json
   def update
     respond_to do |format|
+      PartnerCategory.where(partnering_organization_id: @partnering_organization.id).delete_all
```

Comment:
Not sure if this is the best way to do it @lbkchen

Reply:
This one seems ok